software/apache/README.md

```diff
@@ -25,171 +21,144 @@ This guide walks you through setting up Apache natively on your MacOS with the a
-    $ sudo apachectl stop
-    $ sudo launchctl unload -w /System/Library/LaunchDaemons/org.apache.httpd.plist 2>/dev/null
+
+    which apachectl
+    sudo apachectl stop
+    sudo launchctl unload -w /System/Library/LaunchDaemons/org.apache.httpd.plist 2>/dev/null
 
 Install latest Apache from homebrew
 
-    $ brew install httpd
+    brew install httpd
 
 Configure Apache to auto start using Homebrew services
 
-    $ sudo brew services start httpd
+    brew services start httpd
 
   Useful commands to know:
 
-    $ sudo apachectl start
-    $ sudo apachectl stop
-    $ sudo apachectl -k restart
+    sudo apachectl start
+    sudo apachectl stop
+    sudo apachectl -k restart
 
   Important file locations:
 
     /usr/local/etc/httpd/httpd.conf
 
-## Installing PHP
-
-    $ brew install php@5.6
-    $ brew install php@7.0
-    $ brew install php@7.1
-    $ brew install php@7.2
-
-Important file locations:
-
-    /usr/local/etc/php/5.6/php.ini
-    /usr/local/etc/php/7.0/php.ini
-
 ## Setting up Apache
 
-Choose a "web root". For example, a good place to put web files would be in your home directory under `www`: 
-
-    ~/www/
-
-Create a default VHost
-
-In your web root:
-
-    $ echo "Hello there" > index.html
-    $ echo "<?php phpinfo();" > info.php
+Edit your Apache Configuration File (`/usr/local/etc/httpd/httpd.conf`)
+to enable:
 
-Edit your Apache Configuration File, DocumentRoot to the following. (Replace the existing document root with your document root):
+### Mod Rewrite
+Uncomment out the line
 
-    DocumentRoot "/Users/<your_username>/www"
+    LoadModule rewrite_module libexec/mod_rewrite.so
 
-    <Directory "/Users/<your_username>/www">
-    Options Indexes FollowSymLinks
-    AllowOverride All
-    Require all granted
-    </Directory>
+### vHost file support
 
-Add
+Add the the following line to the end of your `httpd.conf`
 
     Include /usr/local/etc/httpd/vhosts/*.conf
 
-User/Group
-
-    User _www
-    Group _www
-
-### Mod Rewrite
-
-Uncomment the line:
-
-    LoadModule rewrite_module libexec/mod_rewrite.so
-
-
-### Turning on PHP in Apache.
+In your local apache `/usr/local/etc/httpd/` directory run the following commands.
 
-Open the Apache Config file (`/usr/local/etc/httpd/httpd.conf`)
+    mkdir vhosts
+    cd vhosts
+    touch <project_name>.conf
 
-Below the `rewrite_module` line, add the following:
+Restart apache with `brew services restart httpd`
 
-    LoadModule php5_module /usr/local/opt/php@5.6/lib/httpd/modules/libphp5.so
-    #LoadModule php7_module /usr/local/opt/php@7.0/lib/httpd/modules/libphp7.so
-    #LoadModule php7_module /usr/local/opt/php@7.1/lib/httpd/modules/libphp7.so
-    #LoadModule php7_module /usr/local/opt/php@7.2/lib/httpd/modules/libphp7.so
+## Installing PHP
 
-**Edit the following lines:**
+The following script will install each version of PHP, with FPM, configure
+their listening ports for 90## (9071, for example), and configure
+FPM to start using brew services.
```

Comment:
Need to clarify that the port number will align with the version of php when using this install script. So PHP5.6 will listen on port `9056` while PHP7.1 will listen on port `9071`. Took me a few minutes to figure this out when setting up the vhost.